src/main/drivers/pwm_output.c

```diff
@@ -135,7 +136,9 @@ static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 
 void pwmWriteMotor(uint8_t index, uint16_t value)
 {
-    pwmWritePtr(index, value);
+    if (motors[index].enabled) {
+        pwmWritePtr(index, value);
+    }
```

Comment:
I did think that. I thought about simply preventing pwmMotors to be enabled full stop.